Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/n-body/nbody.py", line 9, in <module>
    import sys 
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
